P202-Training.Domain/UsersService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using AutoMapper;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain.Models;
+
+namespace P202.Training.Domain
+{
+    public class UsersService : IUsersService
+    {
+        public void CreateUser(User users, IUserRepository userRepository)
+        {
+            if (users == null) return;
+            var mapUser = Mapper.Map<Data.Entities.User>(users);
```

Comment:
You're still using the Mapper static class. Is there any IMapper interface that you can inject to this class? If not perhaps you can create one that acts as a wrapper to the Mapper static class